src/Microsoft.Azure.WebJobs.Host/Executors/StorageAccountProviderExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static Task<IStorageAccount> GetDashboardAccountAsync(this IStorageAccoun
                 throw new ArgumentNullException("provider");
             }
 
-            return provider.GetAccountAsync(ConnectionStringNames.Dashboard, cancellationToken);
+            return provider.GetAccountAsync(ConnectionStringNames.Dashboard, StorageAccountTypes.GeneralPurpose, cancellationToken);
```

Comment:
I agree, in the past I've gotten flak for not having cancellation token be the last parameter so I did it this way this time.  I'm all for cleaner code so I'll use a default.